sklearn/tests/test_naive_bayes.py

```diff
@@ -918,3 +921,12 @@ def test_n_features_deprecation(Estimator):
 
     with pytest.warns(FutureWarning, match="`n_features_` was deprecated"):
         est.n_features_
+
+
+@pytest.mark.parametrize("Estimator", ALL_NAIVE_BAYES_CLASSES)
+def test_predict_joint_proba(Estimator):
+    est = Estimator().fit(X2, y2)
+    jll = est.predict_joint_log_proba(X2)
+    log_prob_x = logsumexp(jll, axis=1)
+    log_prob_x_y = jll - np.atleast_2d(log_prob_x).T
+    assert_allclose(log_prob_x_y, est.predict_log_proba(X2), 8)
```

Comment:
Ooops, thanks for catching this. I simply changed the name without giving any thought to the signature. :-) 

On my machine the values match _exactly_ (the test repeats the same computation as the tested code, there is no other way), but since in principle we can have probabilities close to 1 (log prob close to 0), I added `atol=1e-7`. This still should catch potential calculation or implementation errors.

Reply:
> I added `atol=1e-7`

Does this make sense?